src/test/java/org/jboss/xavier/integrations/route/MainRouteBuilder_S3Test.java

```diff
@@ -28,14 +28,14 @@
     @Test
     @Ignore
     public void s3Test() throws Exception {
```

Comment:
Well this test is here because it was handy to test the S3 component. We can remove it , but I wanted to leave it there in order to have the code if we need to test more things with the component.